Week01/Problem05/klundquist_05.py

```diff
@@ -0,0 +1,20 @@
+
+num = 1
+incr = 1
+j = 1
+while j <= 20: 
+    found = False
+    k = 1
+    while found == False:
+        found = True
+        num = incr * k
+        i = 1
+        while  i <= j:
+            if num % i != 0:
+                found = False
+                break
+            i = i + 1
+        k = k + 1
+    print(str(j) + " " + str(num))
```

Comment:
I think that [string formatting]() could help you out here. Python is capable of both [C-Style](https://www.learnpython.org/en/String_Formatting) and strictly [Python-3 style](https://docs.python.org/2/library/string.html#custom-string-formatting) string formatting, which both have their [respective advantages](https://pyformat.info/).

For instance, your `print()` statement here could be written as either:
```
# C-Style; use d or i equivalently for integer types
>>> print('%d %d' % (j, num))

# Py3 style;
>>> print('{0} {1}'.format(j, num))
```